src/main/java/zendo/games/zenlib/ZenMain.java

```diff
@@ -79,7 +81,9 @@ public void create() {
         windowCamera.update();
 
         zenAssets = createAssets();
+        loadVisUI();
```

Comment:
Let's move the UI loading code into `ZenAssets` so the pattern stays the same here and we don't need to make loading the ui a separate step in `ZenMain`.

Should be able to move the code from `loadVisUI()` into `ZenAssets.update()` before the `initCachedAssets()` call.